### PR TITLE
Slovak Translation #2

### PR DESCRIPTION
**Pull request by jarole**
added folder values-sk, 3 files translated. Minor fix of the first pull request.

### CODE REVIEW

**Comment by erciccione** [comment]
@fero-sk could you review this translation if you have time pls?
maybe also @MaxXor?

**Comment by m2049r** [comment]
@fero-sk @MaxXor @jarole @erciccione are we good here?

**Comment by MaxXor** [comment]
I don't speak Slovak but I've got a good friend last time who helped out. I'll see if he can check this too.

**Comment by MaxXor** [comment]
> I don't speak Slovak but I've got a good friend last time who helped out. I'll see if he can check this too.

He is on holidays for the next two weeks. I guess you'll have to find someone else to check this.

**Comment by m2049r** [comment]
@MaxXor I think we can wait 2 weeks :) thanks for the help!

**Comment by m2049r** [comment]
@MaxXor so ... :)

**Review by rrunner7 — APPROVED** [review]
Slovak text translations are ok

**Comment by jalso** [comment]
how can i correct typos in these files?

**Comment by m2049r** [comment]
@jalso you either (1) fork @jarole's fork, fix and PR to their fork. or (2), do a review of the PR noting all the errors (click the blue plus which appears next to the next to the line when you look at "files changed"). or (3), if it's not a lot, tell us what needs fixing here in a comment and either @jarole can fix it or I can fix it after merging. or (4) wait for me to merge it, fork this repo and make the changes **on a branch** and make a PR. although (1) is probably the best way to do it in general, i would advise against it in this case, because @jarole made the changes to master and not a seperate branch (e.g. "```translation_slovak```).

**Comment by jarole** [comment]
@jalso , just note in the text (with the blue "plus") where the typo is, or write/describe here in comments, because the translation is 2 months old, I will update the translation with new words (and fix the typos you mention) soon. 
PS: Ďakujem za pomoc chlapi :-) :1st_place_medal:

**Comment by jalso** [comment]
thats all.

**Comment by jalso** [comment]
aj na inom mieste to bolo tak, ze "cely blockchain", ale neviem sa k tomu odborne vyjadrit, nie som jazykovedec.:)

File: app/src/main/res/values-sk/strings.xml
Comment (m2049r) [inline app/src/main/res/values-sk/strings.xml]: this was removed in master - please remove it here.

**Review by m2049r — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-sk/strings.xml
Comment (jarole) [reply app/src/main/res/values-sk/strings.xml]: Thank you @m2049r  for review. 
I assume, that you mean just one string 
 `<string name="language"> `
It has been removed already. Sorry for any inconveniences caused.

**Review by m2049r — APPROVED** [review]

**Comment by m2049r** [comment]
thank you!

**Pull request merged**
